sway/commands/gaps.c

```diff
@@ -68,15 +43,19 @@ static struct cmd_results *gaps_set_defaults(int argc, char **argv) {
 		return cmd_results_new(CMD_INVALID, "gaps",
 				"Expected 'gaps inner|outer <px>'");
 	}
-	if (amount < 0) {
-		amount = 0;
-	}
-
 	if (inner) {
-		config->gaps_inner = amount;
+		config->gaps_inner = (amount >= 0) ? amount : 0;
 	} else {
 		config->gaps_outer = amount;
 	}
+
+    // Prevent negative outer gaps from moving windows out of the container.
```

Comment:
This comment should probably say workspace instead of container.

Reply:
Done